packages/atomic/src/components/common/interface/i18n.ts

```diff
@@ -61,5 +61,12 @@ export function initi18n(atomicInterface: BaseAtomicInterface<AnyEngineType>) {
     interpolation: {
       escape: (str) => DOMPurify.sanitize(str),
     },
+    compatibilityJSON: 'v3',
```

Comment:
If I understand correctly from i18n doc, this is what makes plural support the same as the version currently used on master, is that correct ?

Reply:
Yes